tests/planner_test/tpch.planner.sql

```diff
@@ -146,21 +192,29 @@ order by
 limit 10;
 
 /*
-Projection: [l_orderkey, sum((l_extendedprice * (1 - l_discount))), o_orderdate, o_shippriority] (cost=63701.88)
-  TopN: limit=10, offset=0, orderby=[sum((l_extendedprice * (1 - l_discount))) desc, o_orderdate asc] (cost=63647.88)
-    Aggregate: [sum((l_extendedprice * (1 - l_discount)))], groupby=[l_orderkey, o_orderdate, o_shippriority] (cost=62224.105)
-      Projection: [o_orderdate, o_shippriority, l_orderkey, l_extendedprice, l_discount] (cost=59744.105)
-        HashJoin: inner, on=([o_orderkey] = [l_orderkey]) (cost=55264.105)
-          Projection: [o_orderkey, o_orderdate, o_shippriority] (cost=24811.05)
-            HashJoin: inner, on=([c_custkey] = [o_custkey]) (cost=22091.05)
-              Projection: [c_custkey] (cost=2940)
-                Filter: (c_mktsegment = 'BUILDING') (cost=2700)
-                  Scan: customer[c_custkey, c_mktsegment] (cost=2000)
-              Filter: (1995-03-15 > o_orderdate) (cost=7500)
-                Scan: orders[o_orderkey, o_custkey, o_orderdate, o_shippriority] (cost=4000)
-          Projection: [l_orderkey, l_extendedprice, l_discount] (cost=10220)
-            Filter: (l_shipdate > 1995-03-15) (cost=7500)
-              Scan: lineitem[l_orderkey, l_extendedprice, l_discount, l_shipdate] (cost=4000)
+Projection { exprs: [ l_orderkey, sum, o_orderdate, o_shippriority ], cost: 63701.88 }                            
+└── TopN { limit: 10, offset: 0, order_by: [ desc, asc ], cost: 63647.88 }                                        
+    └── Aggregate { aggs: [ sum ], group_by: [ l_orderkey, o_orderdate, o_shippriority ], cost: 62224.105 }       
```

Comment:
The inner content of `sum` and `asc`/`desc` is still missing. 👀

Reply:
Should be fixed now